distribution/sender/routes/database-event-reader.xml

```diff
@@ -0,0 +1,90 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="db-event-reader" errorHandlerRef="deadLetterChannelBuilder">
+        <from uri="scheduler:reader?initialDelay=60000&amp;delay=60000" />
```

Comment:
1 minute is a little too long for a initialDelay, I think set initialDelay to 30sec, 1 minute is okay for delay